app/src/main/java/com/github/fribourgsdp/radio/FirestoreDatabase.kt

```diff
@@ -50,6 +50,8 @@ class FirestoreDatabase(var refMake: FirestoreRef) : Database {
 
                     val pl = Playlist(playlist["playlistName"]!!,Genre.valueOf(playlist["genre"]!!))
                     pl.id = playlist["playlistId"]!!
+                    pl.savedOnline = true
+                    pl.savedLocally = false

```

Comment:
Is the getUser function that contains this code only called when the user signs in with Google? What I am trying to understand is, is it possible that a user saves a playlist locally, then logs into Google and getUser is called, and his local playlist is overriden by the newly loaded playlists?